database/bfgd/postgres/postgres.go

```diff
@@ -965,6 +965,62 @@ func (p *pgdb) AccessPublicKeyDelete(ctx context.Context, publicKey *bfgd.Access
 	return nil
 }
 
+// BtcBlocksHeightsWithNoChildren returns the heights of blocks that have no
+// child blocks in our database, these represent possible forks that we have
+// not handled yet
+func (p *pgdb) BtcBlocksHeightsWithNoChildren(ctx context.Context) ([]uint64, error) {
+	log.Tracef("BtcBlocksHeightsWithNoChildren")
+	defer log.Tracef("BtcBlocksHeightsWithNoChildren exit")
+
+	/*
+		give me all heights for btc blocks
+		where there does not exist a child of this block
+		and there are no other blocks at this height with children
+		exclude the tip because it won't have children by definition
+	*/
```

Comment:
I think this comment could be improved, additionally I think `//` should be used instead of C-style comments:
```suggestion
	// Query all heights from btc_blocks where the block does not have any
	// children and there are no other blocks at the same height with children.
	// Excludes the tip because it will not have any children.
```

Reply:
ah so @joshuasing I was trying to align each line in the comment with a line in the query, perhaps I should add comments within the query itself? 🤔

Reply:
I think the a comment like this above the query is sufficient and makes it easy to understand what the query is doing without the need to read the query.

I think with the consistency changes suggested in the following comment, the query is also readable enough for people to understand without the need for extra comments. https://github.com/hemilabs/heminetwork/pull/131#discussion_r1618894550